include/roboteam_ai/bt/Node.hpp

```diff
@@ -4,6 +4,7 @@
 #include <vector>
 #include <cstdio>
 #include <cstdarg>
+#include <include/roboteam_ai/world/Field.h>
```

Comment:
This is very bad for compile time: everytime you make a change in field (and the dependencies of field) you have to recompile every skill, condition and tree.

Reply:
As you do a fwd declaration you don't have to include it here.
you can safely include it in de .cpp file.